### PR TITLE
Finished project-news-site

### PR DESCRIPTION
**Pull request by themisk84**
My netlify link
https://sharp-murdock-feb5e6.netlify.app/#Music

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,273 @@
...
+  color: black;
+}
+
+a:hover {
+  color: white;
+  font-size: large;
+
+}
+article > h1 {
+  color: black;
```
Comment (camekman) [inline code/style.css:57]: I really enjoyed the colorfulness of the "Event" and "Music" boxes. A suggestion for the text color would be to use a darker grey to make the text easier to read on some of the colored boxes.

**Review by camekman — APPROVED** [review]
Great job on your project! 
Your code has a nice structure which makes it easy and clear to understand. Your project ticks all the requirement boxes and you have added some very nice hover effects. I think that the styling is great, it goes so well with your theme and creates a cool experience of your page. You should be extra proud of your menu bar that redirect us to the different places on the site. A really nice touch!

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,106 @@
...
+      </div>
+      <div class="music m3">
+        <img src="./andy-stott-never-cover.jpeg" alt="andy-stott-cover" class="music-photo">
+        <p>A noir-pop milestone for the Manchester artist.</p>
+      </div>
+      <div class="music m4">
+        <img src="./XNewYork_PcQWOEQ.png" alt="giegling-logo" class="music-photo">
+        <p>Giegling releasing new album.</p>
+      </div>
+    </section>
```
[the remark this answers is not in the document]
Comment (themisk84) [reply code/index.html:95]: Thank you Camilla!

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,273 @@
...
+}
+ 
+.nav-menu {
+  padding: 5px;
+  color: black;
+}
+
+a:hover {
+  color: white;
+  font-size: large;
```
[the remark this answers is not in the document]
Comment (themisk84) [reply code/style.css:53]: You are right!I will have in mind for the next time!

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,273 @@
...
+  article > h1 {
+    text-align: center;
+  }
+
+ .all-events, .all-music {
+   flex-wrap: wrap;
+   flex-direction: row;
+ }
+
+ @keyframes scaling {
```
[the remark this answers is not in the document]
Comment (themisk84) [reply code/style.css:171]: Yes :)

**Pull request closed without merging**